### PR TITLE
Try State Hook for Bounties

### PR DESCRIPTION
**Pull request by dharjeezy**
Part of: https://github.com/paritytech/polkadot-sdk/issues/239

Polkadot address: 12GyGD3QhT4i2JJpNzvMf96sxxBLWymz4RdGCxRH5Rj5agKW

### CODE REVIEW

**Comment by dharjeezy** [comment]
Hello @liamaharon is there any possibility of getting this reviewed?

File: substrate/frame/bounties/src/lib.rs [hunk 1]
```diff
@@ -807,6 +807,70 @@ pub mod pallet {
...
+		);
+		Ok(())
+	}
+
+	fn get_bounties_count() -> u32 {
+		let mut count = 0;
+		for _ in <Bounties<T, I>>::iter() {
+			count += 1;
+		}
+		count as u32
```
Comment (bkchr) [inline substrate/frame/bounties/src/lib.rs:864]:
```suggestion
		Bounties::<T, I>::iter().count() as u32
```

File: substrate/frame/bounties/src/lib.rs [hunk 2]
```diff
@@ -807,6 +807,70 @@ pub mod pallet {
...
+	}
+
+	fn get_bounties_description_count() -> u32 {
+		let mut count = 0;
+		for _ in <BountyDescriptions<T, I>>::iter() {
+			count += 1;
+		}
+		count as u32
```
Comment (bkchr) [inline substrate/frame/bounties/src/lib.rs:872]:
```suggestion
		BountyDescriptions::<T, I>>::iter().count() as u32
```

**Review by bkchr — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]
/tip medium

**Comment by dharjeezy** [comment]
> /tip medium

Thank you for the tip @kianenigma but looks like the tip bot didn't get triggered

**Pull request merged**